src/test/scenario-test.ts

```diff
@@ -49,7 +49,7 @@ require('source-map-support').install();
         let msg = "";
         let finalResult = true;
         for (const each of folders) {
-            if (each != 'testserver') {
```

Comment:
> if (each != 'testserver') { [](start = 11, length = 28)

not run test server case in ci now.

Reply:
Just add some tests for your issue. and I don't think we can be sure this issue is fixed in the future if we don't add some baseline test. What if some other changes revert it back?

Reply:
As Kaihui are working on add more swagger for testserver, the baseline will be changed frequently currently. We will add it back once Kaihui complete the test server work. ETA is 1 month

---
In reply to: [448835496](https://github.com/Azure/autorest.az/pull/465#discussion_r448835496) [](ancestors = 448835496)